cmd/collector/app/metrics.go

```diff
@@ -28,6 +29,25 @@ const (
 
 	// otherServices is the catch-all label when number of services exceeds maxServiceNames
 	otherServices = "other-services"
+
+	samplerTypeKey           = "sampler_type"
+	samplerTypeConst         = "const"
+	samplerTypeRemote        = "remote"
```

Comment:
remote is an "abstract" sampler type and it will never be reported.

Reply:
So what's returned for adaptive sampling in `(s *Span) GetSamplerType()`?

Reply:
`probabilistic` or `lowerbound`